static/js/admin.js

```diff
@@ -516,6 +518,9 @@ function _setup_page() {
         if (e.which === 13) {
             e.preventDefault();
```

Comment:
Not that it really matters, but I think our convention is to do these prevent/stop calls at the bottom of the handler, not the top.